tests/data/geolife_long/trips.csv

```diff
@@ -1,15 +1,15 @@
-id,user_id,started_at,finished_at,origin_staypoint_id,destination_staypoint_id
-0,0,2008-10-23 02:53:04+00:00,2008-10-23 03:05:00+00:00,,0.0
-1,0,2008-10-23 04:08:07+00:00,2008-10-23 04:34:37+00:00,0.0,1.0
-2,0,2008-10-23 09:42:25+00:00,2008-10-23 10:45:41+00:00,1.0,5.0
-3,0,2008-10-23 11:09:22+00:00,2008-10-23 11:10:57+00:00,5.0,6.0
-4,0,2008-10-24 02:09:59+00:00,2008-10-24 02:47:06+00:00,6.0,
-5,1,2008-10-23 05:53:05+00:00,2008-10-23 06:01:42+00:00,,10.0
-6,1,2008-10-23 10:32:53+00:00,2008-10-23 11:10:19+00:00,10.0,11.0
-7,1,2008-10-23 11:49:08+00:00,2008-10-23 11:49:48+00:00,11.0,12.0
-8,1,2008-10-23 23:43:46+00:00,2008-10-24 00:23:03+00:00,12.0,14.0
-9,1,2008-10-24 01:45:41+00:00,2008-10-24 02:02:46+00:00,14.0,15.0
-10,1,2008-10-24 02:28:19+00:00,2008-10-24 02:31:32+00:00,15.0,16.0
-11,1,2008-10-24 03:16:35+00:00,2008-10-24 04:12:50+00:00,16.0,20.0
-12,1,2008-10-24 05:28:05+00:00,2008-10-24 05:39:50+00:00,20.0,21.0
-13,1,2008-10-24 06:08:42+00:00,2008-10-24 06:35:50+00:00,21.0,
+id,user_id,started_at,finished_at,origin_staypoint_id,destination_staypoint_id,geom
+0,0,2008-10-23 02:53:04+00:00,2008-10-23 03:05:00+00:00,,0.0,"MULTIPOINT (116.3184169999999966 39.9847019999999986, 116.2987250000000046 39.9840190000000035)"
```

Comment:
If the geometry is optional there should be some tests without geometry to see if that works correctly

Reply:
Yes, these tests exist in `io/test_file.py`